localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
It looks like wikipedia calls it "uniform civil number" and  "Unique citizenship number". Would it make sense to use one of those and the English translation of `EIK`? Or do you think using the acronyms in the field names is better?